lib/core-integration/src/Test/Integration/Scenario/CLI/Keys.hs

```diff
@@ -74,17 +74,13 @@ spec  = do
               \e99a00cfed56e9516bc947f327a73e0849882a32a682932c51b42156055abb0b\
               \5d3661deb9064f2d0e03fe85d68070b2fe33b4916059658e28ac7f7f91ca4b12"
 
-        it "bryon keys fail" $ do
-            keyStderr ["child", "--path", "0"] byronKey
-                `shouldReturn`
-                "That extended private key looks \
-                \weird. Is it encrypted? Or is it an old Byron key?\n"
+        it "byron keys does not fail anymore" $ do
+            keySuccess ["child", "--path", "0"] byronKey
+                `shouldReturn` ""
```

Comment:
please double check this change

Reply:
What does "byron keys does not fail anymore" and `shouldReturn ""` mean? 🧐

Reply:
Ok, right, `keySuccess` is new. So this means that the stderr is empty, but you don't check the actually returned key.

Reply:
yeah, improved on this test : https://github.com/input-output-hk/cardano-wallet/pull/1640/commits/014e26b82df20b797eb2187373cc8017a72ad290